.github/workflows/ci_ui.yml

```
@@ -40,3 +40,23 @@ jobs:
+    runs-on: ubuntu-latest
+    env:
+      IMAGE_NAME: reearth/reearth-flow-ui
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Build and Push Docker Image
+        uses: docker/build-push-action@v2
```

actionlint

the runner of "docker/build-push-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)